docs-website/sphinx/apidocs/sdk-v2/lineage-client.rst

```diff
@@ -0,0 +1,6 @@
+Lineage Client
+=======
+
+The DataHub Lineage Client provides a client for searching and retrieving lineage metadata from DataHub.
+
+.. automodule:: datahub.sdk.lineage_client
```

Comment:
the bulleted list having it's own box looks strange

<img width="770" alt="image" src="https://github.com/user-attachments/assets/1a6c8e82-3844-475d-9f1b-0df229591be2" />